account_invoice_pricelist/views/account_invoice_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_account_invoice_customer_form" model="ir.ui.view">
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <group col="4" colspan="4">
+                    <field name="pricelist_id"/>
+                    <button type="object"
```

Comment:
IMO this button shouldn't be included, but auto-recompute on pricelist change + product/qty change, the same as standard Odoo does in sales orders.

Reply:
In pratical use cases, you may stick on manual values because of one shot product invoicing. I use it in v8, then I need of this button for v10 too.

Reply:
I prefer the auto thing and following v8 paradigm.

Reply:
You prefer, ok but do you use this module ?

Reply:
> IMO this button shouldn't be included, but auto-recompute on pricelist change + product/qty change, the same as standard Odoo does in sales orders.

Seems it's changed, because in oldest version, the mechanism was not implemented on sale orders.
I think that : 
* both approaches are correct;
* OCA should adapt its module to odoo behaviour. If sale orders now works with onchange on pricelist, we should implement it to improve user experience.

so :+1: for @pedrobaeza proposal.

regards.

Reply:
ok 2 against 1, you know what to do @rvalyi ;-)

Reply:
Ah, right, I mixed fiscal position with pricelist. Indeed, prices are not changed on that case. Well, OK then.

Reply:
This view code breaks the invoice form view on v10... the labels of other fields of the right column disappear. I wonder if someone really uses the code in real life... :-(

Reply:
Going back to PR https://github.com/OCA/account-invoicing/pull/342  At least, that one didn't break views !!!

Reply:
Propose a PR for solving it. Don't make a drama of this, and it's not breaking. It's only UI.

Reply:
I don't make a drama of this. I just say that this work is not tested.